controllers/openshift_controller.go

```diff
@@ -71,6 +71,7 @@ const (
 	DEFAULT_PEER_PODS                   = "10"
 	peerpodConfigCrdName                = "peerpodconfig-openshift"
 	peerpodsMachineConfigPathLocation   = "/config/peerpods"
+	peerpodsImageJobsPathLocation       = "/config/peerpods"
```

Comment:
This will end up mixing machine config and job YAML files under the same directory, whose path suggests is was originally introduced for machine configs (see 6577693479bc0f181442923e93e5e2c2df0fa271). Maybe use a different path for the jobs ?

Why putting the image jobs at the same location as the machine config ?